docs/content/docs/build/containers.md

```diff
@@ -149,8 +149,9 @@ You should always check the `isDirty` flag before closing the container or navig
 
 A container is considered dirty in the following cases:
 
-1. The container has local changes that have not yet been acknowledged by the service. These unacknowledged changes will be lost if the container is closed.
-1. There is no network connection while making changes to the container. These changes cannot be acknowledged by the service until the network connection is restored.
+1. The container has been created in the detached state, and it has not been attached yet.
```

Comment:
Given that (I think) we expose tri-state (attached, attaching, detached) I think this description has to talk about attaching state. In particular - that container is dirty in this state, and that if container is closed, host may never know if file was created or not.

Reply:
I can call out "attaching" state here explicitly as part of 1. The message here should probably still be concise and boil down to, if container is not  in "attached" state (whether detached or attaching), it is "dirty".